trusted_applet/main.go

```diff
@@ -230,7 +230,32 @@ func runWithNetworking(ctx context.Context) error {
 		return ctx.Err()
 	}
 
-	time.Sleep(5 * time.Second)
+	updateFetcher, updateClient, err := updater(ctx)
+	if err != nil {
+		return fmt.Errorf("failed to create updater: %v", err)
+	}
+
+	// TODO(al): figure out where & how frequently we should be doing this.
+	// For now, since we're still developing/testing this, we'll be very aggressive
+	// checking for and installing updates.
+	go func() {
+		for {
+			select {
+			case <-time.After(5 * time.Second):
```

Comment:
Consider using a ticker here instead. ISTR that time.After causes more garbage but I don't have time to go check my citation right now.